.github/dependabot.yml

```diff
@@ -60,13 +62,17 @@ updates:
     ignore:
       - dependency-name: "go.opentelemetry.io/*"
       - dependency-name: "github.com/open-telemetry/*"
+      - dependency-name: "github.com/openshift/*"
+        versions: ["v3.9.0+incompatible"]
     schedule:
       interval: "weekly"
   - package-ecosystem: "gomod"
     directory: "/pkg/signalfx-agent"
     ignore:
       - dependency-name: "go.opentelemetry.io/*"
       - dependency-name: "github.com/open-telemetry/*"
+      - dependency-name: "github.com/openshift/*"
+        versions: ["v3.9.0+incompatible"]
```

Comment:
Should these changes be added to the 'gendependabot' target in the Makefile instead?

Reply:
oh right, forgot about that...

Reply:
Shall I add a simple github job to fail if this file is manually changed?

Reply:
Updated but I'm not sure how platform agnostic the changes are. imo we should adopt a templated approach in future changes.